sqlobject/src/test/java/org/jdbi/v3/sqlobject/TestPositionalBinder.java

```diff
@@ -47,38 +49,30 @@ public void setUp() throws Exception {
     @Test
     public void testOnePositionalParameter() {
         String name = somethingDao.findNameById(2);
-        assertEquals("Keith", name);
+        assertThat(name).isEqualTo("Keith");
     }
 
     @Test
     public void testManyPositionalParameters() {
         Integer id = somethingDao.getIdByNameAndCode("Coda", 14);
-        assertEquals(3, id.intValue());
+        assertThat(id).isEqualTo(3);
     }
 
     @Test
     public void testInsertWithPositionalParameters() {
         somethingDao.insertSomething(4, "Dave", 90);
 
         List<Map<String, Object>> rows = handle.select("select * from something where something_id=?", 4);
-        assertEquals(rows.size(), 1);
-
-        Map<String, Object> row = rows.get(0);
-        assertEquals(row.get("something_id"), 4);
-        assertEquals(row.get("name"), "Dave");
-        assertEquals(row.get("code"), 90);
+        assertThat(rows).containsExactlyElementsOf(ImmutableList.of(
+                ImmutableMap.of("something_id", 4, "name", "Dave", "code", 90)));
```

Comment:
The compiler gives me warning "unchecked generic array creation for varargs parameter" when I use `containsExactly` with maps.